redbeams/src/main/java/com/sequenceiq/redbeams/flow/redbeams/start/RedbeamsStartFlowConfing.java

```diff
@@ -23,12 +25,12 @@
 public class RedbeamsStartFlowConfing extends AbstractFlowConfiguration<RedbeamsStartState, RedbeamsStartEvent>
         implements RetryableFlowConfiguration<RedbeamsStartEvent> {
 
-    private static final RedbeamsStartEvent[] REDBEAMS_INIT_EVENTS = {REDBEAMS_START_EVENT};
+    private static final RedbeamsStartEvent[] REDBEAMS_INIT_EVENTS = {REDBEAMS_CERT_REPLACE_EVENT};
 
     private static final List<Transition<RedbeamsStartState, RedbeamsStartEvent>> TRANSITIONS =
             new Transition.Builder<RedbeamsStartState, RedbeamsStartEvent>().defaultFailureEvent(REDBEAMS_START_FAILED_EVENT)
-                    .from(INIT_STATE).to(START_DATABASE_SERVER_STATE).event(REDBEAMS_START_EVENT)
-                    .defaultFailureEvent()
+                    .from(INIT_STATE).to(CERT_REPLACE_STATE).event(REDBEAMS_CERT_REPLACE_EVENT).defaultFailureEvent()
```

Comment:
As per above, please keep `START_DATABASE_SERVER_STATE` the init state (and `REDBEAMS_START_EVENT` the init event), and add `CERT_REPLACE_STATE` after the start has been executed.